pkg/lightning/backend/backend.go

```diff
@@ -143,6 +143,10 @@ type AbstractBackend interface {
 
 	OpenEngine(ctx context.Context, config *EngineConfig, engineUUID uuid.UUID) error
 
+	SetEngineTSIfNotExists(ctx context.Context, engineUUID uuid.UUID, ts uint64) error
```

Comment:
suggestion: 

1. turn this function into `AllocateTSIfNotExists(ctx context.Context, engineUUID uuid.UUID) error`, which is the same as `allocateCommitTS` followed by `SetEngineTSIfNotExists`
2. local and importer backends can share the same `allocateCommitTS` implementation, and both contain the `pdAddr` field
3. tidb and noop backends just return nil.
4. get rid of `rc.isImporterBackend()`.

Reply:
done in https://github.com/pingcap/br/pull/1101/commits/7716e864c6abfecf7b266d0f655da97b220c461a